nvflare/fuel/hci/tools/admin.py

```diff
@@ -38,8 +40,9 @@ def main():
 
     try:
         os.chdir(args.workspace)
-        workspace = os.path.join(args.workspace, "startup")
-        conf = FLAdminClientStarterConfigurator(app_root=workspace, admin_config_file_name=args.fed_admin)
+        workspace_dir = os.path.join(args.workspace, "startup")
+        workspace = Workspace(root_dir=args.workspace)
```

Comment:
suggest use `workspace.get_startup_kit_dir()`

Reply:
Will address this in another PR